src/SingleGame.jsx

```diff
@@ -48,10 +52,17 @@ function SingleGame() {
           created at: {createDate(singleReview.created_at)}
         </p>
         <p className="single-cat">category: {singleReview.category}</p>
-        <button className="single-button">👎</button>
-        <button className="single-button">👍</button>
+        <button className="single-button" onClick={voteChangeDown}>
+          👎
+        </button>
+        <button className="single-button" onClick={voteChangeUp}>
+          👍
+        </button>
+        <p>
+          <strong className="votes-up">{votesUp}</strong>
+        </p>
         <p>
-          <strong>{singleReview.votes}</strong>
+          <strong className="votes-down">{votesDown}</strong>
```

Comment:
The strong HTML element should only be used to highlight important information on a page, I'd suggest making this bold using CSS instead 👍